module/users/helper.js

```diff
@@ -1261,8 +1273,57 @@ module.exports = class UsersHelper {
       }
     });
   }
+
+  /**
+   * Find getRootOrganisationOfUser.
+   * @method
+   * @name getRootOrganisationOfUser
+   * @param {String} userToken - user token
+   * @param {String} userId - userId
+   * @returns {Array} - user rootOrganisation data
+   */
+  static getRootOrganisationOfUser(
+    userToken,
+    userId
+  ) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let result = [];
+        // Fetch user profile
+        let userProfile = await userService.profile(userToken, userId);
```

Comment:
@VISHNUDAS-tunerlabs we can directly call userService.profile from anywhere in service why this function is required?

Reply:
@ankitpws we have to  use same checks in other private program creation block.  using this function we can avoid using same checks and code in different places